services/drupal/composer.json

```diff
@@ -246,6 +247,7 @@
         "drupal/module_filter": "3.x-dev",
         "drupal/page_manager": "^4.0@beta",
         "drupal/paragraphs": "^1.9",
+        "drupal/paragraphs_entity_embed": "2.0-alpha2",
```

Comment:
@azinck Looks like the work you did for the embed spike made it into this PR. Is it okay if I revert these commits in this branch?